src/routes/OverviewPage/DescriptionsCard.tsx

```diff
@@ -32,7 +30,7 @@ function DescriptionsCard() {
   }
 
   const panelContent = (
-    <DrawerPanelContent widths={{ xl: 'width_50' }}>
+    <DrawerPanelContent widths={{ xl: 'width_50' }} className="pf-v6-u-m-xs">
```

Comment:
The extra little bit of margin prevents the drawer panel from covering up the border of the containing Card, which looks buggy.